libs/barista-components/sunburst-chart/README.md

```diff
@@ -47,10 +48,10 @@ attribute is required to create a valid output.
 
 #### Methods
 
-| Name           | Params                              | Description                         |
-| -------------- | ----------------------------------- | ----------------------------------- |
-| `openOverlay`  | `node: DtSunburstChartTooltipNode>` | Open an overlay for the given node. |
-| `closeOverlay` |                                     | Closes the overlay if open.         |
+| Name           | Params                              | Description                                     |
+| -------------- | ----------------------------------- | ----------------------------------------------- |
+| `openOverlay`  | `node: DtSunburstChartTooltipNode>` | Deprecated. Open an overlay for the given node. |
```

Comment:
Why deprecate this method? It seems rather useful to have a possibility to open the overlay programmatically.